hkmc2/shared/src/test/mlscript/handlers/StackSafety.mls

```diff
@@ -130,21 +131,21 @@ sum(10000)
 //│   return prev
 //│ };
 //│ sum1 = function sum(n) {
-//│   let scrut, tmp, tmp1, curDepth, stackDelayRes, getLocals2, Cont$func$sum$StackSafety$_mls_L118_4_57$1;
+//│   let scrut, tmp, tmp1, curDepth, stackDelayRes, getLocals2, Cont$func$sum$StackSafety$_mls_L119_4_57$1;
 //│   getLocals2 = function getLocals() {
 //│     let n1, prev, thisInfo, tmp2;
 //│     prev = getLocals1();
-//│     n1 = new runtime.LocalVarInfo.class("n", "697", n);
+//│     n1 = new runtime.LocalVarInfo.class("n", "699", n);
```

Comment:
Q: Why are these numbers changing all the time?

Reply:
It is the uid of the symbol of the local var

Reply:
Why do we need  this?

Reply:
I added it in case the LSP found it useful, i.e. so you can jump to a certain variable's definition site by clicking it in the sidebar. But I think we mentioned that this is fine because you need to add a certain flag for all the debug stuff to show (it should be hidden in this file).

Reply:
Well you can include the location instead for that purpose anyway? And during editing the uid will change a lot depending on the surrounding while the location might still be useful if there is some smart way of updating the location based on the text inserted.